Created Products API

SidJTolentino opened the pull request:
- Created controller and routes for Products
- Adjusted the prisma model of Product

Issue: The app crashes when I delete a product ID.  I needed to re-run the server.

SidJTolentino committed:
Created Products API

- Created controller and routes for Products
- Adjusted the prisma model of Product

nikkoroque reviewed (changes requested):
Revisit and fix delete method

SidJTolentino committed:
Fixed Product Controller

- Fixed the Delete method in the product controller.
- No longer display "app crash" message when a product is deleted

nikkoroque committed:
Merge branch 'dev' into feature/api-products
